static/themes/dark.css

```diff
@@ -10,7 +10,7 @@ html, body {
 }
 
 body {
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-family: "Open Sans", "Clear Sans", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
Could you please show me the difference? After you add these two font.

Reply:
The first problem is, that [light theme](https://github.com/marktext/marktext/blob/216c5226c29f0ce04ec36f511d0bd0de7e472af9/static/themes/light.css#L14) and [dark theme](https://github.com/marktext/marktext/blob/216c5226c29f0ce04ec36f511d0bd0de7e472af9/static/themes/dark.css#L13) are inconsistent. The second, the width of the dialog is too small. That's why I increased dialog width to `454px`

> Could you please show me the difference? After you add these two font

What difference did you mean?

Reply:
Sorry my bad, I forget add "Open Sans" and  "Clear Sans" to dark.css...


I thought you added these two fonts. 😹